raiden-ts/package.json

```diff
@@ -44,6 +44,7 @@
     "@types/events": "^3.0.0",
     "@types/isomorphic-fetch": "0.0.35",
     "@types/jest": "^24.0.25",
+    "@types/lodash": "^4.14.149",
```

Comment:
This is going to fail. It was in dependencies because some SDK module is exporting a Memoized function, and if the type is not accessible, it fails. You must either keep it in dependencies or find where this Memoized export is, and cast it to the normal function type, to prevent the emitted `.d.ts` depending on the lodash's types.

Reply:
Ok will revert

Reply:
I found it: `transport/utils.ts` and `utils/types.ts`. You can safely cast it to the normal function type, as the "memoized" part of the type isn't used (just at runtime).

Reply:
Lets do it in another PR with more context